docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -47,7 +47,7 @@ Similarly, when assembling to JSON, the `--set-parameters` option will cause any
 
 As described earlier, the authoring tools are designed to work well in a CI/CD environment where changes are made in a pipeline by people with different responsibilities and authority.  In this setting, changes to documents can trigger changes downstream, e.g. the editing of a control would cause an update in the catalog, which could then flow down to an updated SSP.  These changes can occur automatically via actions that restrict the potential changes to the generated documents.  Examples are the `--set-parameters` option on the `-assemble` tools, and both `--required-sections` and `allowed-sections` for `profile-assemble`.  If a document change triggers an assemble action, changes to parameters can only occur if the action has `--set-parameters` in the command.  Similarly, `profile-assemble` will fail if the sections do not meet the requirements specified in the command options.  Another feature of the `-assemble` tools is that they won't create a new OSCAL file if the output already exists and its content would not be changed.  This prevents undesired triggering of downstream actions when there is no actual change in content.
 
-There is a standalone demonstration of the SSP generation process with trestle in the [Trestle SSP Demo](https://github.com/oscal-compass/compliance-trestle-ssp-demo) that captures the entire process of SSP authoring: from creation of a component definition from CSV file to a final formatted system security plan in Word (.docx) format.
+There is a standalone demonstration of the SSP generation process with trestle in the [Trestle SSP Demo](https://github.com/IBM/compliance-trestle-ssp-demo) that captures the entire process of SSP authoring: from creation of a component definition from CSV file to a final formatted system security plan in Word (.docx) format.
```

Comment:
Should not this repo be moved from IBM to oscal-compass? If so, then this change should not be made, and the repo should be moved.

Reply:
@degenaro i've opened an issue #1688 as this will take time for IBM approvals etc. I would suggest we finish this task / PR first.